Library/Formula/bazel.rb

```diff
@@ -23,13 +23,7 @@ def install
     system "./compile.sh"
     system "./output/bazel", "build", "scripts:bash_completion"
 
-    (prefix/"base_workspace").mkdir
-    cp_r Dir["base_workspace/*"], (prefix/"base_workspace"), :dereference_root => true
-    bin.install "output/bazel" => "bazel"
```

Comment:
This `bin.install` line needs to remain, or else nothing will be installed in `/usr/local/bin`.